numpy/lib/twodim_base.py

```diff
@@ -705,7 +705,9 @@ def histogram2d(x, y, bins=10, range=None, normed=None, weights=None,
     >>> x = np.random.normal(2, 1, 100)
     >>> y = np.random.normal(1, 1, 100)
     >>> H, xedges, yedges = np.histogram2d(x, y, bins=(xedges, yedges))
-    >>> H = H.T  # Let each row list bins with common y range.
+    >>> # Histogram does not follow Cartesian convention (see Notes),
+    >>> # therefore transpose H for visualization purposes.
+    >>> H = H.T
```

Comment:
Just a very minor suggestion, but you can pull the explanatory text outside of executable prompts to make it stand out a little more:

```suggestion

    Histogram does not follow Cartesian convention (see Notes),
    therefore transpose H for visualization purposes.
    
    >>> H = H.T
```

One thing to be aware for this is that there must be blank lines between the executable prompts and the text.